<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>16</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="get">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>680</y>
      <width>309</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Get image</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Center (required)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="locationType">
         <item>
          <property name="text">
           <string>Classic string</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KABC Address</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KABC Geo</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KLineEdit" name="locationString">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="KLineEdit" name="locationCity"/>
       </item>
       <item>
        <widget class="KLineEdit" name="locationStreet"/>
       </item>
       <item>
        <widget class="QSpinBox" name="locationNumber"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="locationLatitude">
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="locationLongitude">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Zoom (required)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="zoom">
       <property name="maximum">
        <number>21</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Size (required)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="width">
         <property name="maximum">
          <number>640</number>
         </property>
         <property name="value">
          <number>640</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string comment="X axis">x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="height">
         <property name="maximum">
          <number>640</number>
         </property>
         <property name="value">
          <number>640</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Scale (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="scale">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Format (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="format">
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gif</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpg-baseline</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Map type (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="map">
       <item>
        <property name="text">
         <string>roadmap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>satellite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>terrain</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hybrid</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Sensor (required)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="sensor">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string comment="A boolean value">false</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="A boolean value">true</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Marker (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="KLineEdit" name="markerLabel">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="markerSize">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string comment="Marker size">Tiny</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Marker size">Small</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Marker size">Middle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Marker size">Normal</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="KColorButton" name="markerColor">
         <property name="color">
          <color>
           <red>0</red>
           <green>0</green>
           <blue>192</blue>
          </color>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KLineEdit" name="markerLocation"/>
     </item>
     <item>
      <widget class="QPushButton" name="addMarker">
       <property name="text">
        <string>Add marker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="markers"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string comment="Action that quits the application">Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
